### ISSUE TITLE
possible replication

### ISSUE THREAD

**Issue by pbellec** [brainhack-school2020/stephaniealley_bhs2020_project#3]
As you are planning to look at age prediction in fMRI, it would be fantastic if you could try to replicate the classic Andrews-Hanna study (Fig 1 and 4 in particular):
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2709284/

**Comment by stephaniealley** [comment]
Yes, I think this would be a great way to focus the project. Thanks for the suggestion!

**Comment by stephaniealley** [comment]
I have started going through the Andrews-Hanna paper, and I have a question regarding the type of dataset I should use to replicate the study. In the paper I see that the dataset consisted of 93 adults aged 18 to 93 because the study explores the nature of functional disruptions resulting in cognitive decline in normal aging.

My initial idea for the age prediction was to work with a dataset containing children and adults. While I am not particularly proficient in this type of study, I would imagine that it should be possible to identify functional differences between children and adults. I am not sure, however, that such a dataset would be appropriate to use to replicate the study. Would it be better to find a more comparable dataset that contains only adults over a wide age range?

**Comment by pbellec** [comment]
@stephaniealley I had not understood you were looking at developmental data. Then it wouldn't work for the replication I suggested. Lots of previous works have looked at developmental changes in rs-fMRI, and it is definitely of interest. If you already found a dataset you could work with, I don't think it makes sense to try and find a new dataset just for including the replication I mentioned.
